projects/plugins/social/src/class-meta-tags.php

```diff
@@ -342,8 +342,33 @@ public function render_tags( $post = null ) {
 			);
 		}
 
-		// Add SIG image if enabled.
-		$tags = apply_filters( 'jetpack_open_graph_tags', $tags );
+		/**
+		 * Filter the minimum width of the images used in Jetpack Open Graph Meta Tags.
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param int 200 Minimum image width used in Jetpack Open Graph Meta Tags.
+		 */
+		$image_width = absint( apply_filters( 'jetpack_open_graph_image_width', 200 ) );
+
+		/**
+		 * Filter the minimum height of the images used in Jetpack Open Graph Meta Tags.
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param int 200 Minimum image height used in Jetpack Open Graph Meta Tags.
+		 */
+		$image_height = absint( apply_filters( 'jetpack_open_graph_image_height', 200 ) );
+
+		/**
```

Comment:
I've realised this shouldn't really be here. I think we normally [refer to the docs](https://github.com/search?q=repo%3AAutomattic%2Fjetpack+%22this+filter+is+documented%22&type=code) where it's defined originally, [like this](https://github.com/Automattic/jetpack/blob/e60ed56953a7f7285160b39883359462dd78fb9b/projects/plugins/social/src/class-meta-tags.php#L121).

We could do that here and do the same for the other filters above too.

Reply:
Done. Thanks